charts/victoria-metrics-distributed/README.md

```diff
@@ -0,0 +1,209 @@
+4. vmauth-global-read: global query entrypoint, proxies requests to one of the zone `vnauth-read-proxy` with `first_available` policy.
+5. grafana(optional): uses `vmauth-global-read` as default datasource.
+
+>Note:
+As the topology shown above, this chart doesn't include components like vmalert, alertmanager, etc by default.
+You can install them using dependency [victoria-metrics-k8s-stack](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack) or having separate release.
+
+### Why use `victoria-metrics-distributed` chart?
+
+One of the best practice of running production kubernetes cluster is running with [multiple availability zones](https://kubernetes.io/docs/setup/best-practices/multiple-zones/). And apart from kubernetes control plane components, we also want to spread our application pods on multiple zones, to continue serving even if zone outage happens. 
+
+VictoriaMetrics supports [data replication](https://docs.victoriametrics.com/cluster-victoriametrics/#replication-and-data-safety) natively which can guarantees data availability when part of the vmstorage instances failed. But it doesn't works well if vmstorage instances are spread on multiple availability zones, since data replication could be stored on single availability zone, which will be lost when zone outage happens.   
+To avoid this, vmcluster must be installed on multiple availability zones and each of them should contain complete data. In this way, as long as one zone is available, both global write and read entrypoint work without interruption.
+
+### How to write data?
+
+The chart provides `vmauth-global-write` as global write entrypoint, it supports [push-based data ingestion protocols](https://docs.victoriametrics.com/vmagent/#how-to-push-data-to-vmagent) as VictoriaMetrics does.
+You can use this entrypoint as vmagent's remoteWrite url or use other clients push data to it directly.
+
+### How to query data?
+
+The chart provides `vmauth-global-read` as global read entrypoint, it picks the first zone as it's preferred datasource and switches automatically to next zone if first one is unavailable, check [vmauth `first_available`](https://docs.victoriametrics.com/vmauth/#high-availability) for more details.  
+If you have workloads like vmalert or grafana which deployed on one zone specifically, you can use `vmauth-read-proxy` on the same zone as your datasource, since it always prefer "local" vmcluster which reduces cross-zone traffic. 
+
+You can also pick other proxies like kubernetes service which supports [Topology Aware Routing](https://kubernetes.io/docs/concepts/services-networking/topology-aware-routing/) as global read entrypoint.
+
+### What will happen if one zone outraged?
+
+If availability zone `zone-eu-1` outraged, `vmauth-global-write` and `vmauth-global-read` still work without interruption, and following things happen:
```

Comment:
```suggestion
### What will happen in case of zone outage?

In case availability zone `zone-eu-1` is not available, `vmauth-global-write` and `vmauth-global-read` still work without interruption, and following things happen:
```

Reply:
Most likely "outrage" was used due to a typo :)

Reply:
yes, thanks!